src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java

```diff
@@ -333,11 +340,19 @@ public boolean validateAndProcessDataImport(ODMContainer odmContainer, String st
                                     dataImportReport.setItemGroupRepeatKey(itemGroupDataBean.getItemGroupRepeatKey());
                                     dataImportReport.setItemOID(itemDataBean.getItemOID());
                                     dataImportReports.add(dataImportReport);
-                                    continue;
                                 } else if (itemObject instanceof Item) {
                                     item = (Item) itemObject;
                                 }
-
+                                for(DiscrepancyNoteBean discrepancyNoteBean : itemDataBean.getDiscrepancyNotes().getDiscrepancyNotes()){
+                                    try {
+                                        importValidationService.validateQuery(discrepancyNoteBean);
+                                        createQuery(discrepancyNoteBean, tenantStudy, studySubject, eventCrf, itemDataBean.getItemOID(), Integer.parseInt(itemGroupDataBean.getItemGroupRepeatKey()));
+                                    }catch (Exception e){
```

Comment:
You should only catch the OpenClinicaSystemException since that's what you are expecting. All runtime exceptions should be just thrown and maybe caught at a higher level if needed.